tests/validation/tests/v3_api/test_auth.py

```diff
@@ -610,7 +610,7 @@ def enable_ad(username, token, enable_url=CATTLE_AUTH_ENABLE_URL,
         "groupSearchAttribute": "sAMAccountName",
         "nestedGroupMembershipEnabled": nested,
         "port": PORT,
-        "servers": [HOSTNAME_OR_IP_ADDRESS],
+        "servers": [HOSTNAME_OR_IP_ADDRESS, HOSTNAME_OR_IP_ADDRESS],
```

Comment:
this looks like duplicated data, is this line correct? I would expect this to take a new variable that is a list itself or something of the sort, instead of the same variable 2x

Reply:
Yes, this line is intended like this: the API should accept a list of more than one server. And I did not want to test any error cases when that host is not reachable. I was not brave enough to replace the first list entry with a wrong servername because my local dapper ci tests don't execute that test. And I have no idea how to enable that.